src/diffpy/utils/diffraction_objects.py

```diff
@@ -331,29 +331,29 @@ def uuid(self):
     def uuid(self, _):
         raise AttributeError(_setter_wmsg("uuid"))
 
-    def get_array_index(self, value, xtype=None):
+    def get_array_index(self, xtype, xvalue):
```

Comment:
Switched the order for the parameters based on behavior:

user first decide which xtype to search for, and then enter the value second..